server/amp/handlers/story-page.js

```diff
@@ -153,14 +153,24 @@ async function ampStoryPageHandler(
     if (ampHtml instanceof Error) return next(ampHtml);
 
     res.set("Content-Type", "text/html");
+
+    const pageCacheKey = `page/${config["publisher-id"]}/story-page`;
+    let cacheKeys = [storyToCacheKey(config["publisher-id"], story), pageCacheKey];
+
+    const getAdditionalCacheKeys = get(opts, ["featureConfig", "getAdditionalCacheKeys"]);
```

Comment:
could you also pls add documentation for `getAdditionalCacheKeys` in amp lib?

Reply:
done